bindata/v4.1.0/kube-apiserver/check-endpoint-config-cm.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-kube-apiserver
+  name: check-endpoint-config
+data:
+  config.yaml: |
+    apiVersion: operator.openshift.io/v1
+    kind: GenericOperatorConfig
+    leaderElection:
+      disable: true
```

Comment:
this manifest surprises me. Where is it needed?

Reply:
The library-go based controllers  have leader election enable by default, limiting control loops to only one per namespace. I need this config to disable the leader election logic.

Reply:
> The library-go based controllers have leader election enable by default, limiting control loops to only one per namespace. I need this config to disable the leader election logic.

update library-go to have a `WithoutLeaderElection`

Reply:
@sttts I've updated the [`ControllerCommandOptions`](https://github.com/openshift/library-go/pull/811) to achieve the same result without the extra resources.